api-service/src/types/MetricModel.ts

```diff
@@ -0,0 +1,14 @@
+export interface Labels {
+    entity: string,
+    id: string,
+    dataset_id: string | null,
+    endpoint: string,
+    status: number,
+    request_size: number,
+    response_size: string | number | string[] | undefined
```

Comment:
why response_size is the not number ?

Reply:
There is a possibility that **res.getHeader('content-length')** method may return above types, but it's rare. Yesterday while testing all methods and cases, the number was returned.